v2/discoveryv1/discovery_v1.go

```diff
@@ -4722,10 +4722,7 @@ func UnmarshalAggregationResult(m map[string]json.RawMessage, result interface{}
 	obj := new(AggregationResult)
 	err = core.UnmarshalPrimitive(m, "key", &obj.Key)
 	if err != nil {
-		err = core.UnmarshalPrimitive(m, "key_as_string", &obj.Key)
-		if err != nil {
-			return
-		}
```

Comment:
oh just curious why we remove this part out?

Reply:
That's a great catch. That needs to be re-added as a hand edit.

Reply:
Okay. I will update the review to `request changes`